src/main/java/org/opentripplanner/openstreetmap/model/OSMNode.java

```diff
@@ -34,13 +38,13 @@ public boolean hasCrossingTrafficLight() {
     return hasTag("crossing") && "traffic_signals".equals(getTag("crossing"));
   }
 
-  /**
-   * Checks if this node is bollard
+  /* Checks if this node is a barrier which prevents motor vehicle traffic
    *
    * @return true if it is
    */
-  public boolean isBollard() {
-    return isTag("barrier", "bollard");
+  public boolean isMotorVehicleBarrier() {
+    var barrier = this.getTag("barrier");
```

Comment:
It's too late now because I don't want to hold up the merge of this PR but whenever we touch this code again, there is the following new helper method for checking if it's one of a set of tag values:

https://github.com/opentripplanner/OpenTripPlanner/blob/f664f770402e1006f491b9cbd18e3d9724019c80/src/main/java/org/opentripplanner/openstreetmap/model/OSMWithTags.java#L230-L235

Reply:
Oh yes, I forgot that new method. Let's fix this in some upcoming OSM related PR.